### ISSUE TITLE
Lessons navigation (next/prev links) not working on more than 10 lessons per section.

### ISSUE THREAD

**Issue by blancoms** [gocodebox/lifterlms#1323]
The next/prev navigation lessons are not working properly when you have 10 or more lessons in one section.

This is how it is ordered right now:
https://github.com/gocodebox/lifterlms/blob/b839ca0f6413c7b249d93ab7f059e09729e8e859/includes/models/model.llms.lesson.php#L756-L764

You should use named meta queries like this:
```php
				'llms_lesson_order' => array(
					'key'     => '_llms_order',
					'value'   => $next_position,
					'compare' => '>=',
					'type'    => 'numeric',
				),
			),
			'orderby'        => 'llms_lesson_order',
			'order'          => 'ASC',
```

**Comment by thomasplevy** [comment]
@blancoms 

This sounds like a duplicate of #1316 

Can you confirm you're seeing this issue on the latest version of LifterLMS?

**Comment by blancoms** [comment]
> @blancoms
> 
> This sounds like a duplicate of #1316
> 
> Can you confirm you're seeing this issue on the latest version of LifterLMS?

You're right, it's a dup of #1316. It's already fixed in 4.4.2. Thanks!
